tests/eval/good/uint_conversions.scilexp

```diff
@@ -11,4 +11,17 @@ let res4 = builtin to_uint32 big_num in   (* None *)
 let res5 = builtin to_uint64 big_num2 in   (* Some big_num2 *)
 let res6 = builtin to_uint64 minus_one in   (* None *)
 
+let zero = 0x00 in
+let one_1 = 0x01 in
+let one_3 = 0x000001 in
+let one_4 = 0x00000001 in
+let one_8 = 0x0000000000000001 in
+
+let res7 = builtin to_uint32 zero in
+let res8 = builtin to_uint32 one_1 in
+let res9 = builtin to_uint32 one_3 in
+let res10 = builtin to_uint32 one_4 in
+let res11 = builtin to_uint256 one_1 in
+let res11 = builtin to_uint256 one_8 in
```

Comment:
As far as I understand, this PR adds `to_uint64` / `to_uint128` (and the signed versions too). If this is correct, could we have those tested too, please?

Reply:
Added now for other unsigned ones. I didn't add the builtin itself for signed versions. I'm not convinced we need them. If we do add, we do sign extend if the input bystrx is smaller in size? Not sure. That's why I've left it for now.